src/hooks/useAddressHistory.ts

```diff
@@ -64,7 +65,13 @@ export function useAddressHistory() {
   };
 
   useEffect(() => {
-    id && addAddress(id);
+    const isIncorrectAddress = !isAddressEmpty && !jettonAddress;
+    const isEmptyAddress =
+      (!jettonAddress && location.pathname === ROUTES.jetton + "/") ||
```

Comment:
can we specify in react-router instead that it only routes to the jetton page if a param was specified? it seems strange that we have to make the empty check